docs/overview.md

```diff
@@ -6,35 +6,34 @@ and match modular elements as desired. Along side, all modules are permissioned
 and sandboxed to isolate modules for greater application security.
 
 For more explanation please see the [standard
-library](https://github.com/tendermint/basecoin/blob/unstable/docs/quark/stdlib.md)
+library](stdlib.md)
 and
-[glossary](https://github.com/tendermint/basecoin/blob/unstable/docs/quark/glossary.md)
+[glossary](glossary.md)
 documentation.
 
 For a more interconnected schematics see these
-[framework](https://github.com/tendermint/basecoin/blob/unstable/docs/graphics/overview-framework.png)
+[framework](../graphics/overview-framework.png)
 and
-[security](https://github.com/tendermint/basecoin/blob/unstable/docs/graphics/overview-security.png)
+[security](../graphics/overview-security.png)
 overviews.
 
 ## Framework Overview
 
-### Transaction (tx) 
+### Transactions
 
-Each tx passes through the middleware stack which can be defined uniquely by
-each application.  From the multiple layers of tx, each middleware may strip
-off one level, like an onion.  As so, the transaction must be constructed to
-mirror the execution stack, and each middleware should allow embedding an
-arbitrary tx for the next layer in the stack.
+Each transaction passes through the middleware stack which can be defined
+uniquely by each application. From the multiple layers of tx, each middleware
+may strip off one level, like an onion. As such, the transaction must be
+constructed to mirror the execution stack, and each middleware module should
+allow embedding an arbitrary tx for the next layer in the stack.
```

Comment:
This paragraph is nice, I initially struggled to understand why we had all these `middleware`, this paragraph reminds me of the clarity I got after @ethanfrey explained the concept to me a few times, while we were working on basecoin.